src/syntethic-options-watcher.h

```diff
@@ -1,26 +0,0 @@
-// License: Apache 2.0. See LICENSE file in root directory.
-// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
-#pragma once
```

Comment:
Not needed anymore?

Reply:
It was accidentally pushed. Name is misspelled.